spec/graphql/mutations/memberships/revoke_spec.rb

```diff
@@ -23,22 +23,21 @@
   it_behaves_like "requires permission", "organization:members:update"
 
   it "Revokes a membership" do
-    user = create(:user)
-    create(:membership, organization:, role: :admin, user:)
+    user_to_remove = create(:membership, organization:, role: :admin)
```

Comment:
We could avoid some confusion with

```suggestion
    membership_to_remove = create(:membership, organization:, role: :admin)
```

😅

Reply:
Yes! It's confusing enough 😶